docs/management/security/encryption.md

```diff
@@ -121,11 +121,11 @@ Sentinel will be configured with `tls-port` if and only if `tls-replication` is
 
 Additional TLS configuration is available to control the choice of TLS protocol
 versions, ciphers and cipher suites, etc. Please consult the self documented
-`redis.conf` for more information.
+`valkey.conf` for more information.
 
 ### Performance considerations
 
-TLS adds a layer to the communication stack with overheads due to writing/reading to/from an SSL connection, encryption/decryption and integrity checks. Consequently, using TLS results in a decrease of the achievable throughput per Redis instance (for more information refer to this [discussion](https://github.com/redis/redis/issues/7595)). 
+TLS adds a layer to the communication stack with overheads due to writing/reading to/from an SSL connection, encryption/decryption and integrity checks. Consequently, using TLS results in a decrease of the achievable throughput per Valkey instance (for more information refer to this PLEASE_REVIEW{[discussion](https://github.com/redis/redis/issues/7595)}). 
```

Comment:
This looks like ancient history - 4 years ago and a major version. I would drop the parenthetical.

WDYT @madolson?

Reply:
Agree. Let's remove that. Alternatively read that issue and write any relevant info in the docs and remove the link.